cbxtools/cli.py

```
@@ -3,979 +3,1144 @@
+        return 1
+    # Process archives
+    try:
+        start_time = time.time()
+        # Call the conversion function
+        from .conversion import process_archive_files
+
+        (
+            success_count,
+            total_original_size,
+            total_new_size,
+            processed_files,
```

Ruff (0.13.1)

Unpacked variable `processed_files` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)